### ISSUE TITLE
Fetching the most recent entry from the performance array

### ISSUE THREAD

**Issue by gokatz** [ebidel/appmetrics.js#7]
if the user has made more than one measurement for the given event. Can we pop and return the last entered performance entry?

**Comment by ebidel** [comment]
Good call. Or maybe you should always get an array?

**Comment by gokatz** [comment]
yeah... We should, I think. Users will be able to get the `duration` array at any time without having to fetch the duration property every time while measuring a metric. How about supporting both `getDurationArray` and `getRecentDuration` ??
